ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,21 +84,76 @@ development team of the Linux Project
 
 - Security Standards
 - License Compliance Policies
-- Tooling
-  - Build in-house
-  - Adopt existing tools
-      - Commercial tools
-      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
-- Design OSS infrastructure
-
-### OS Community Education
 
-#### Create Internal OS Ambassadors
 
-- Drive event sponsorship
-- Train speakers for coding forums
-  
-#### Create educational programs
+### 📝 Establish and Improve Open Source Policies and Processess
+
+##### Policies
+
+- Using and compliance policy
```

Comment:
- Usage of open source to build SaaS offering

Reply:
@CsatariGergely I thought about writing each type of deliverable out but keep things simple I wrote "Mixing code (owned and third-party OSS/proprietary) and distributing as products or services" and the SaaS use case I see as the "products or services" part

Reply:
I would separate the SaaS and the delivery case as they are fundamentally different from most of the open source licenses point of view.

Reply:
@CsatariGergely I just expanded the 'Comply with open source license obligations' under the 'Oversee Open Source Compliance' to cover various delivery methods including SaaS. Welcome your feedback.